tests/ctest/ar_generator.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
```

Comment:
License for these files should be 0BSD?

Reply:
I will change all the licenses in a later PR.